cylc/flow/cfgspec/globalcfg.py

```diff
@@ -131,6 +122,23 @@
             Conf('abort on inactivity', VDR.V_BOOLEAN)
             Conf('abort on stalled', VDR.V_BOOLEAN)
 
+        with Conf('mail', desc='''
+            Options for email handling.
+        '''):
```

Comment:
Doesn't the proposal call for this to go under `[scheduling]` instead of `[cylc]`?

Reply:
Yes. the `[cylc]` should become `[scheduler]`. I wasn't going to add that to this change, which turned out to be fiddly enough on its own.

Reply:
Oh, so there's going to be both `[scheduling]` and `[scheduler]`...

Reply:
Looking at the proposal 
https://cylc.github.io/cylc-admin/proposal-config-changes.html#updated-global-config just `scheduler`, but it's quite possible that the doc I've been looking at is out of date.